stable/airflow/UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrading Steps
 
+## `v7.8.X` → `v7.9.0`
+
```

Comment:
Please add a descrption opf the new feature:

```md
__The following IMPROVEMENTS have been made:__
* You can now give the airflow ServiceAccount GET/LIST on event Resources
   * This is needed for `KubernetesPodOperator(log_events_on_failure=True)`
   * To enable, set `rbac.events` to `true` (Default: `false`)

__The following values have been ADDED:__
* `rbac.events`
```

Reply:
👍